digiwf-integrations/digiwf-email-integration/digiwf-email-integration-service/src/main/resources/application.yml

```
@@ -5,7 +5,7 @@ spring:
     name: @project.artifactId@
   cloud:
     function:
-      definition: functionRouter;sendMessage;
+      definition: functionRouter;sendMessage;sendMailFromEventBus;
```

@lmoesle das war wahrscheinlich eher aus versehen oder?
Gibt es ja nur im example und nicht im service

---

```suggestion
      definition: functionRouter;sendMessage;
```

Ich habe das am Freitag schon in Openshift händisch gefixed und im Dev Branch auch. Das gehört weg, war mein Fehler